Feature/5 add production deployment

bjtulinl opened the pull request:
- For now, the app can be started in two ways:
  1. debug mode: python3 app_flair.py 
  2. server: gunicorn app_flair:app -c gunicorn.conf

bjtulinl committed:
#5 - Add production deployment
- I set the number of workers and threads of gunicorn as the options. You can run the application like "python3 app_flair.py --workers 2 --threads 2"(default number is 1)
- I also attach the json file of the request.

bjtulinl committed:
#5 - Add production deployment
- For now, we can start the app with two ways: debug mode: python3 app_flair.py (--pos pos --ner ner)(default) or server with the config file: gunicorn app_flair:app -c gunicorn.conf

jcklie commented:
It works for me. Can you please update the readme with these commands to run it?

jcklie reviewed (approved):
Works for me, nice!

bjtulinl committed:
Update README.md

Add instructions on how to start the application on
1. debug mode
2. a gunicorn server.

bjtulinl committed:
Delete request.json
